mylyn.docs/wikitext/.project

```diff
@@ -14,4 +14,15 @@
 	<natures>
 		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
What is the goal of this filteredResources? First time I see it so I'm just curious.

Reply:
The filteredResources are used to avoid duplicate Projects in a Workspace.

Here an Example with a  project test that includes a project TestBundle1 within a folder bundle
<img width="165" alt="Bildschirmfoto 2023-12-02 um 07 44 05" src="https://github.com/eclipse-mylyn/org.eclipse.mylyn/assets/13169360/f84a531c-59d3-4674-a350-b8381d0f6688">

For example, you will receive duplicate results when searching in the workspace.

Reply:
We have filterdResources in many projects to remove the target folder from the ProjectExplorer view.
I think I have found all the places where we have a grouping parent folder and add an additional filter there for folders with a .project file.

Please let me know if you find a duplicate project.